libr/core/cmd_anal.c

```diff
@@ -4862,7 +4862,10 @@ static void cmd_anal_esil(RCore *core, const char *input) {
 				while (pc < end) {
 					left = R_MIN (end - pc, 32);
 					r_asm_set_pc (core->assembler, pc);
-					ret = r_anal_op (core->anal, &op, addr, buf, left, R_ANAL_OP_MASK_ESIL); // read overflow
+					ret = r_anal_op (core->anal, &op, pc, buf + pc - bb->addr, left, R_ANAL_OP_MASK_ESIL); // read overflow
```

Comment:
add R_ANAL_OP_MASK_HINT

Reply:
> .

@radare: i fear your change breaks the thing again...

Reply:
`op.esil` always is the empty string

Reply:
ah sry, forget what i said... i replaced R_ANAL_OP_MASK_ESIL with R_ANAL_OP_MASK_HINT instead of adding it..